pulumi/folio/index.ts

```diff
@@ -9,10 +9,12 @@ import * as kafka from "./kafka";
 import * as postgresql from "./postgresql";
 import * as folio from "./folio";
 import * as util from "./util";
+import * as prometheus from "./prometheus";
```

Comment:
Is this referring to the file that was removed?

Reply:
Yes let me fix that.

Reply:
There were a few other things I missed. It's gone now.